index.html

```diff
@@ -859,194 +859,180 @@
+makes it easier for an automated system to store data about that [=person=].
+Examples of [=identifiers=] for a [=person=] can be:
+* their name,
+* an identification number including those mapping to a device that this
+[=person=] may be using,
+* their phone number,
+* their location data,
+* an online identifier such as email or IP addresses,
+* browser [=fingerprinting|fingerprints=] (based on a combination of
+configuration characteristics), or
+* factors specific to their physical, physiological, genetic, mental,
+economic,
+cultural, social, or behavioral [=identity=],
+* strings derived from [=identifiers=], for instance through hashing.
+
+<dfn>Cross-context recognition</dfn> is the act of recognising that an
+[=identity=] in one [=context=] is the same [=person=] as an [=identity=] in
+another [=context=]. This contributes to [=surveillance=], [=correlation=],
+and [=identification=].
+
+[=Cross-context recognition=] can at times be [=appropriate=] but web users
+need to be able to control when websites do it as much as possible. Systems
+which [=cross-context recognition|recognize=] people across [=contexts=] need
+to be careful not to apply the [=principles=] of one [=context=] in ways that
+violate the [=principles=] around use of information acquired in a different
+[=context=]. This is particularly true for [=vulnerable=] people, as
+recognising them in different [=contexts=] may force traits that reveal their
+vulnerability into the open. For example, if you meet your therapist at a
+party, you expect them to have different discussion topics with you than they
+usually would, and possibly even to pretend they don't know you.
+
+Note that a UA may not always be powerful enough to prevent [=cross-context
+recognition=]. For example, if two or more services [=inappropriately=]
+require an identifying piece of information in order to use the services.
+
+<dfn>Cross-site recognition</dfn> is when a site determines with high
+probability that a visit to the site comes from the same person as another
+visit to a *different* site.
+
+Traditionally, sites have accomplished this using
+<a href="https://tess.oconnor.cx/2020/10/parties#environment-cross-site">
+cross-site</a> cookies, but it can also be done by having someone navigate to
+a link that has been decorated with an identifier, collecting the same piece
+of identifying information on both sites, or by correlating the timestamps of
+an event that occurs nearly-simultaneously on both sites. A privacy harm
+occurs through [=cross-site recognition=], unless the person could reasonably
+expect the sites to discover this and has effective means to mitigate the issue.
+
+<dfn>Same-site recognition</dfn> is when a single site discovers and uses the
+fact that two or more visits probably came from the same [=person=].
+
+This is generally accomplished by either "supercookies" or browser
+[=fingerprinting=]. A privacy harm occurs if a [=person=] reasonably expects
```

Comment:
super-cookies or fingerprinting also accomplish same-site recognition across clearing of data. jyasskin notes that that is across a partition, even though it's the same site, because a partition includes user-selected clearing. but could maybe clarify that in the text.

Reply:
One approach to this sentence is to front load the second sentence and add 'harm' like:

A privacy harm occurs if a [person](https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/177.html#dfn-individual) reasonably expects that they'll be using a different [identity](https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/177.html#dfn-identity) for different visits to a single site, but the site engages in [same-site recognition](https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/177.html#dfn-same-site-recognition). This harm is generally accomplished by either "supercookies" or browser [fingerprinting](https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/177.html#dfn-fingerprint).